VERSION

```diff
@@ -1 +1 @@
-0.2.1
+0.5.0
```

Comment:
I think the security-services version going forward is supposed to be 1.0.0 for Edinburgh/master

Reply:
ok will change it